homeassistant/components/tuya/aes_cbc.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+"""AES-CBC encryption and decryption for account info."""
```

Comment:
Drop this file and all the encryption. It's not the responsibility of integrations to encrypt their own data.